<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="Utama" native="true">
      <property name="styleSheet">
       <string notr="true">background-color:#DAD7CD;</string>
      </property>
      <widget class="QLabel" name="Class">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>400</y>
         <width>401</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="Camera">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>331</width>
         <height>251</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 5px solid #588157;
font-size: 12px;
border-radius: 10px;</string>
       </property>
       <property name="text">
        <string>Camera</string>
       </property>
      </widget>
      <widget class="QLabel" name="Judul">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>10</y>
         <width>341</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>SISTEM KLASIFIKASI SAMPAH</string>
       </property>
      </widget>
      <widget class="QPushButton" name="testing">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>350</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#344E41;
text-weight: bold;
color :white;
border-radius:10px;</string>
       </property>
       <property name="text">
        <string>Testing</string>
       </property>
      </widget>
      <widget class="QLabel" name="Rmbg">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>90</y>
         <width>161</width>
         <height>101</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 5px solid #588157;
font-size: 12px;
border-radius: 10px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="ambil_gambar">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>350</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#344E41;
text-weight: bold;
color :white;
border-radius:10px;</string>
       </property>
       <property name="text">
        <string>Ambil Gambar</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Result">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>340</y>
         <width>331</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 5px solid #588157;
font-size: 120%;
border-radius: 10px;
padding:5px 5px;
background: transparent;</string>
       </property>
       <property name="placeholderText">
        <string>Hasil prediksi</string>
       </property>
      </widget>
      <widget class="QLabel" name="grayscale">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>220</y>
         <width>161</width>
         <height>101</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 5px solid #588157;
font-size: 12px;
border-radius: 10px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QWidget" name="LeftMenu_3" native="true">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>70</y>
         <width>131</width>
         <height>301</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#3A5A40;
border-radius:15px;</string>
       </property>
       <widget class="QPushButton" name="btn_Training">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#A3B18A;
text-weight: bold;
border-radius:10px;</string>
        </property>
        <property name="text">
         <string>Training</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../training.svg</normaloff>../training.svg</iconset>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_Testing">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>160</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#A3B18A;
text-weight: bold;
border-radius:10px;</string>
        </property>
        <property name="text">
         <string>Testing</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../testing.svg</normaloff>../testing.svg</iconset>
        </property>
       </widget>
       <widget class="QLabel" name="Menu">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>61</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white;
</string>
        </property>
        <property name="text">
         <string>Menu</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_Dataset">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>230</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#A3B18A;
text-weight: bold;
border-radius:10px;</string>
        </property>
        <property name="text">
         <string>Dataset</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../Dataset.svg</normaloff>../Dataset.svg</iconset>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="label_RG">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>70</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Remove Background</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_gray">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>200</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Grayscale</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
